middleware.js

```diff
@@ -302,10 +302,9 @@ async function requireMozAdmin(req, res, next) {
   const user = req.user;
 
   if (!user?.primary_email.includes("@mozilla.com")) {
```

Comment:
Q: Could we use [`.endsWith()`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith) here instead of the more generic/risky `.includes()`?
Might be fractionally more secure.